.gitignore

```diff
@@ -34,3 +34,6 @@ trained_models/
 /mlruns.db-journal
 mlflow_backend.py
 demo4.py
+/.venv
+!time_series_croston.ipynb
```

Comment:
this ipynb should not be in the gitignore. If you do not want to push it, do not add it to the commit.

Reply:
OK, i thought it might be useful for you and @ltsaprounis  to review, but we can remove.